terraform/fdi_lambda.tf

```
@@ -196,16 +196,27 @@ resource "aws_lambda_permission" "fdi_lambda_allow_bucket" {
   source_arn    = data.aws_s3_bucket.findings_data.arn
 }
 
-# Create the notification that triggers our Lambda function to run whenever
-# an object is created in our findings data bucket
+# Create the notification configuration for the findings data bucket
 resource "aws_s3_bucket_notification" "fdi_lambda" {
   bucket = data.aws_s3_bucket.findings_data.id
 
+  # Trigger the appropriate lambda function whenever an object with the configured
```

Keep the capitalization train rolling... 🚂 
```suggestion
  # Trigger the appropriate Lambda function whenever an object with the configured
```